packages/lesswrong/components/common/WarningBanner.tsx

```diff
@@ -9,20 +9,23 @@ const styles = (theme: ThemeType) => ({
     fontFamily: theme.typography.commentStyle.fontFamily,
     color: theme.palette.text.warning,
     fontSize: 14,
-    lineHeight: '18px',
+    lineHeight: '20px',
     fontWeight: '500',
     padding: '10px 8px',
     borderRadius: 4,
     backgroundColor: theme.palette.background.warningTranslucent,
     marginBottom: 8,
   },
   icon: {
-    transform: "translateY(1px)",
+    transform: "translateY(3px)",
```

Comment:
Do the changes to the `translateY` or the `lineHeight` need forum-gating?

Reply:
Hmm I think probably not? I think it's more likely that no one tested it, so I won't bother for now